### PR TITLE
Issue #2496 - Update Use Cases to follow updated model

### PR DESCRIPTION
**Pull request by koekiebox**
# Accounts and Balances BC updated to include the following;

* Updated UC to reflect overall style of ref architecture
* Reference external flows where A+B are being referenced from
* Minor updates to phrasing
* Add cononical model

# Before
image

# After
image
image
image
image
image
image
image
image

### CODE REVIEW

File: docs/refarch/boundedContexts/accountsAndBalances/index.md [hunk 1]
```diff
@@ -1,28 +1,100 @@
 # Accounts and Balances BC
 
-The Accounts and Balances BC acts as the "central ledger" for the system.  It interacts primarily with the Settlements and the Transfers BCs, and is a directed sub-system, which means that it is a dependency of the Settlements and Transfers BCs that use it as a "system of record" for the financial accounting.
+The Accounts and Balances BC acts as the "central ledger" for the system.  It interacts primarily with the Settlements and the Transfers BCs, and is a directed sub-system, which means that it is a dependency of the Settlements and Transfers BCs that use it as a "financial system of record" for the financial accounting.
```
Comment (pedrosousabarreto) [inline docs/refarch/boundedContexts/accountsAndBalances/index.md:3]: ...and the Participants Lifecycle BC

**Review by pedrosousabarreto — COMMENTED** [review]
Looks good to me , only one minor comment

**Review by pedrosousabarreto — APPROVED** [review]
All good

File: docs/refarch/boundedContexts/accountsAndBalances/index.md (hunk 1 above)
Comment (koekiebox) [reply docs/refarch/boundedContexts/accountsAndBalances/index.md]: Resolved.

**Pull request merged**